website/docs/concepts/probes.md

```diff
@@ -4,4 +4,1102 @@ title: Probes
 sidebar_label: Probes
 ---
 
-## Coming soon
+## Introduction
+
+In Litmus, Probes are pluggable checks that can be defined within the ChaosEngine for any Chaos Experiment. The experiment pods execute these checks based on the mode they are defined in & factor their success as necessary conditions in determining the verdict of the experiment (along with the standard `in-built` checks).
```

Comment:
The word `Chaos Experiment` can be linked to the `Experiment` concepts section.

Reply:
The rest is good.

Reply:
Are we going for a `litmus` terminology here? It was decided to follow the `ChaosCenter` terminology and other sections have also done the same :thinking:

Reply:
Litmus is Still the Product, Chaos Center is the wrapper around litmus which has all the components like chaos exporter, event tracker, etc. We are still installing Litmus ChaosCenter, not just ChaosCenter afaik. ChaosCenter and ChaosAgents are both parts of Litmus. That is my understanding.

Reply:
Have added the link to Chaos Engine in the Pre-requisite section. Chaos Experiment CR won't be a pre-requisite for Probes.